e2e/pages/events/manageDocumentsLAEvent.page.js

```diff
@@ -135,6 +141,11 @@ module.exports = {
     I.click(this.fields.supportingDocuments(elementIndex).confidential);
   },
 
+  async selectConfidentialCourtBundle() {
+    const elementIndex = await this.getActiveElementIndex();
```

Comment:
I think that there is an `I.getActiveElementIndex()` method (see the actor.js file) in which case it could replace the current method in here.